src/coreclr/src/jit/hwintrinsiccodegenarm64.cpp

```diff
@@ -854,6 +854,21 @@ void CodeGen::genHWIntrinsic(GenTreeHWIntrinsic* node)
             }
             break;
 
+            case NI_AdvSimd_ReverseElement16:
+                GetEmitter()->emitIns_R_R(ins, emitSize, targetReg, op1Reg,
```

Comment:
This seems like it could be table-driven, but I suspect that it just isn't worth it, given that there are so few of these?

Reply:
I thought about generalizing this, so it could be table-driven, but, as you said, - there little benefit of doing so - since there are only three intrinsics like these.